napalm/base/base.py

```diff
@@ -94,9 +94,14 @@ def _netmiko_open(self, device_type, netmiko_optional_args=None):
         except NetMikoTimeoutException:
             raise ConnectionException("Cannot connect to {}".format(self.hostname))
 
-        # ensure in enable mode if not force disable
-        if not self.force_no_enable:
+        # Disable enable mode if force_no_enable is true (for NAPALM drivers
+        # that support force_no_enable)
+        try:
```

Comment:
Now that I'm looking at this again, it does look a little strange, wondering if the following would be more readable:

```python
if not getattr(self, 'force_no_enable', False):
    self._netmiko_device.enable()
```

Reply:
Any thoughts on this @tynany / @ktbyers ?

Reply:
I think the try/except is fine.

Reply:
This PR looks good to me so I am fine with merging it.